gh-fork-ribbon.css

```diff
@@ -98,11 +92,9 @@
   top: 42px;
   right: -43px;
 
-  -webkit-transform: rotate(45deg);
-  -moz-transform: rotate(45deg);
-  -ms-transform: rotate(45deg);
-  -o-transform: rotate(45deg);
-  transform: rotate(45deg);
+  -webkit-transform: rotate(-45deg);
```

Comment:
Here and elsewhere you're replacing `45deg` with `-45deg`. I assume that's not intentional?